### DISCUSSION TITLE
How would I do circular text using tailwind css?

### DISCUSSION THREAD

**Discussion by godoom** [tailwindlabs/tailwindcss#10673] [Help]
I want to have text links surrounding an image and im using Tailwind css. Kind of like this photo:
![image](https://user-images.githubusercontent.com/35696558/221056605-374dc4ed-b65f-402f-a94b-f0bdd38919e3.png)
. This is a nextjs project

All examples ive seen are using vanilla html, css and javascript

**Comment by wongjn** [comment]
This is outside the realm of CSS, and by extension, Tailwind. You could consider using the [`<textPath>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/textPath) SVG element.

**Comment by dr5hn** [comment]
Here you go!
https://jh3y.medium.com/circular-text-with-css-cf374de94435
https://codepen.io/jh3y/pen/LYBdKXE
